changelogs/fragments/20231219-deploy_flask_app-update-arguments-spec.yml

```diff
@@ -0,0 +1,10 @@
+---
+breaking_changes:
+  - >-
+    roles/deploy_flask_app - Add parameter ``deploy_flask_app_bastion_ssh_private_key`` to define
+    the path to the ssh private key file to use to connect to the bastion host (https://github.com/redhat-cop/cloud.aws_ops/issues/103).
```

Comment:
#103 points to the issue. i think we should point to the PR here.

Reply:
Updated. @GomathiselviS could you please review?

Reply:
@abikouo 
I'll review the pull request by the end of this week. Before approval, I need to verify the deploy_flask_app role. Apologies for the delay. In the meantime, could you please check https://issues.redhat.com/browse/ACA-1022? This Jira task involves creating a playbook for deploying a Flask app in an HA architecture. I'm curious if removing the region parameter from the playbook will impact the requirement.